src/ewig/draw.cpp

```diff
@@ -94,9 +95,9 @@ void draw_text(const buffer& buf, coord size)
             auto hl_first = row == starts.row ? std::max(starts.col, 0) : 0;
             auto hl_last  = row == ends.row   ? std::max(ends.col, 0) : str.size();
             ::addnwstr(str.c_str(), hl_first);
-            ::attron(COLOR_PAIR(color::selection));
+            ::attron(COLOR_PAIR((int)color::selection));
             ::addnwstr(str.c_str() + hl_first, hl_last - hl_first);
-            ::attroff(COLOR_PAIR(color::selection));
+            ::attroff(COLOR_PAIR((int)color::selection));
```

Comment:
What was the error here?

Reply:
Conversion error from the `enum class` to int

Reply:
Makes sense.